ui/pages/swaps/swaps.util.js

```diff
@@ -850,6 +854,25 @@ export const getNetworkNameByChainId = (chainId) => {
   }
 };
 
+/**
+ * @param {string} chainId
+ * @returns string token image url based on networks chainId
+ */
+export const getNetworkImageByChainId = (chainId) => {
```

Comment:
This is the same as the function `getNativeCurrencyImage` in `ui/selectors/selectors.js` (Or, at least, that function has the same inputs and outputs.) So I suggest using the existing `getNativeCurrencyImage` instead of adding this new function.

Reply:
@danjm Fixed in: 2179697b44f506d161e5258c52789f632ae8bda8